test/testcafe/spec/EnrollAuthenticatorPasswordView_spec.js

```diff
@@ -18,11 +18,13 @@ const errorMock = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
   .respond(xhrAuthenticatorEnrollPasswordError, 403);
 
-fixture('Authenticator Enroll Password');
+fixture('Authenticator Enroll Password')
+  .meta('v3', true);
 
 async function setup(t) {
   const enrollPasswordPage = new FactorEnrollPasswordPageObject(t);
   await enrollPasswordPage.navigateToPage();
+  await t.expect(Selector('form').exists).eql(true);
```

Comment:
I think we do this a lot for v3. Feel free to move it into a base object if/when it's re-used.

Reply:
@leemchale-okta tried to add this at the baseform level and found some tests failed since they don't actually show the form at all. Perhaps we can look at a more nuanced inclusion (e.g. flag in the constructor or utility method for setup in the baseform)